HowLong/HowLong.py

```diff
@@ -89,12 +97,12 @@ def run(self):
         process = Process(pid=self.pid,command=self.command)
         readable_command = process.command
         start_time = process.start_time
-        logging.debug("Running " + readable_command)
+        logging.debug(colored("Running " + readable_command), 'green')
         while process.is_running():
             sleep(self.timer_interval)
             elapsed_time = (time() - start_time) * 1000
-            logging.info(red(str(timedelta(milliseconds=elapsed_time))))
-        logging.debug("Finished " + readable_command)
+            logging.info(colored(str(timedelta(milliseconds=elapsed_time)), 'blue'))
+        logging.debug(colored("Finished " + readable_command), 'red')
```

Comment:
This needs to be: `logging.debug(colored("Finished " + readable_command, 'red'))`